### ISSUE TITLE
Not able to access LB rules on port 2000.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#1766]
Server version - v0.32.0-rc2

Create LB rule with listener port to 2000.
Ha proxy is configured as expected and port 2000 is also opened on the host.

But I get the following read error when trying to access the LB rule:

```
Sangeethas-MacBook-Pro:testwget sangeethahariharan1$ wget http://104.197.95.29:2000/name.html
--2015-08-07 17:43:27--  http://104.197.95.29:2000/name.html
Connecting to 104.197.95.29:2000... connected.
HTTP request sent, awaiting response... Read error (Connection reset by peer) in headers.
Retrying.

--2015-08-07 17:44:30--  (try: 2)  http://104.197.95.29:2000/name.html
Connecting to 104.197.95.29:2000... connected.
HTTP request sent, awaiting response... ^C
Sangeethas-MacBook-Pro:testwget sangeethahariharan1$ 
```

root@sangee32rc2-10acre-2:/home/sangeethahariharan1# iptables-save | grep 2000
-A CATTLE_PREROUTING -p tcp -m addrtype --dst-type LOCAL -m tcp --dport 2000 -j DNAT --to-destination 10.42.237.84:2000
root@sangee32rc2-10acre-2:/home/sangeethahariharan1# 
<img width="749" alt="screen shot 2015-08-07 at 5 50 40 pm" src="https://cloud.githubusercontent.com/assets/4266958/9148320/e353c6ca-3d2c-11e5-85fa-e40a59bb2c69.png">

**Comment by alena1108** [comment]
Sangeetha, I can't see this issue on my setup. Can you check yours to see what other apps might be listening on port 2000 on your host?

**Comment by sangeethah** [comment]
This issue is reproducile in GCE instances which were added custom host as well as adding "Digital Ocean"

**Comment by yasker** [comment]
I've taken a look at this issue. The conclusion is Comcast(our internet vendor) blocks (interferes in fact) the port 2000.

1. Same exactly rule works fine with port 2001.
2. tcpdump shows the TCP handshake established, but no http packets passed through after that. 
3. Access the port from inside GCE instance works. I've noticed that the external IP used by GCE is not binding(by ip addr) on the same host. It was routed by GCE's network to the outside of the host, then routed back by some routers to the host. So it's not really the localhost accessing in the case.
4. Sangeetha said the same port on Digital Ocean doesn't work as well, which would be weird for two providers block the same not well known port. 
5. Port 2000 was listed by http://www.speedguide.net/port.php?port=2000 as used by RemoteAnywhere to expose a web server to install some software, and been taken advantage by some trojan software. So it's probably the reason why the http protocol got blocked somewhere along the way.
6. I checked the ip using my phone's 4G network, it works.

So I came to conclusion that it must be our ISP has done something to port 2000, likely block any http protocol on the port. Since the official protocol on port 2000 is Cisco SCCP, which is not a http protocol, it would be mostly fine.
